modules/ROOT/attachments/admin-api.yaml

```diff
@@ -584,6 +584,33 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/partitions_local_summary'
+  /partitions/majority_lost:
+    get: 
+      tags: 
+      - Partitions
+      summary: Get partitions with lost majority
+      description: List of partitions with lost majority given an input set of dead nodes.
+      operationId: majority_lost
+      # responses: 
```

Comment:
Double check - [spec](https://github.com/redpanda-data/redpanda/blob/dev/src/v/redpanda/admin/api-doc/partition.json#L366) indicates "void" but this should return something like an erray?

Reply:
Correct, "void" is a mistake, this endpoint returns an array of `ntp_with_majority_loss`

Reply:
@ztlpn can we update the spec to be accurate please?  This will be important because we are trying to automate Admin API doc generation from the spec moving forward. Thanks

Reply:
Paging @bharathv as the main author :)

Although I would say that problems with keeping schemas for request bodies in sync with the code are bound to appear because they these schemas are not used anywhere (we do separate validation in the admin server code).